macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -26,7 +26,7 @@
 		33CC10F32044A3C60003C045 /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 33CC10F22044A3C60003C045 /* Assets.xcassets */; };
 		33CC10F62044A3C60003C045 /* MainMenu.xib in Resources */ = {isa = PBXBuildFile; fileRef = 33CC10F42044A3C60003C045 /* MainMenu.xib */; };
 		33CC11132044BFA00003C045 /* MainFlutterWindow.swift in Sources */ = {isa = PBXBuildFile; fileRef = 33CC11122044BFA00003C045 /* MainFlutterWindow.swift */; };
-		B4EC64F4BCD762C3F0F55DD5 /* Pods_Runner.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 571FD19C035907C8FDB1CC86 /* Pods_Runner.framework */; };
```

Comment:
make sure your next PR's dont include these files cos it shouldn't if your running it on web